app/views/devise/shared/_links.html.erb

```diff
@@ -3,7 +3,7 @@
 <% end -%>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= !Account.global_tenant? && current_account.allow_signup == "true" ? (link_to t(".sign_up"), new_registration_path(resource_name)) : ' ' %><br />
+  <%= !Account.global_tenant? && current_account.respond_to?(:allow_signup) && current_account.allow_signup == "true" ? (link_to t(".sign_up"), new_registration_path(resource_name)) : ' ' %><br />
```

Comment:
This looks fine to me.  Do you think we should use `try` instead to make it a little more concise?
```suggestion
  <%= !Account.global_tenant? && current_account.try?(:allow_signup) == "true" ? (link_to t(".sign_up"), new_registration_path(resource_name)) : ' ' %><br />
```

Reply:
Yea, i always forget `try` is a thing now that `&` is around. If this works the same, i'm more than happy with that.